### ISSUE TITLE
Can't convert to color: type=0x1

### ISSUE THREAD

**Issue by hushicai** [crazycodeboy/react-native-splash-screen#123]
It seams that something is wrong in android.

react native: 0.47.2
android: 6.0.1

java.lang.UnsupportedOperationException: Can't convert to color: type=0x1
--
2 android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2423)
3 ......
4 Caused by:
5 java.lang.UnsupportedOperationException:Can't convert to color: type=0x1
6 android.content.res.TypedArray.getColor(TypedArray.java:453)
7 com.android.internal.policy.PhoneWindow.generateLayout(PhoneWindow.java:3779)
8 com.android.internal.policy.PhoneWindow.installDecor(PhoneWindow.java:3983)
9 com.android.internal.policy.PhoneWindow.setContentView(PhoneWindow.java:383)
10 android.app.Dialog.setContentView(Dialog.java:515)
11 org.devio.rn.splashscreen.SplashScreen$1.run(SplashScreen.java:32)
12 android.app.Activity.runOnUiThread(Activity.java:5573)
13 org.devio.rn.splashscreen.SplashScreen.show(SplashScreen.java:26)
14 org.devio.rn.splashscreen.SplashScreen.show(SplashScreen.java:47)
15 com.gf.mobile.clickeggs2.MainActivity.onCreate(MainActivity.java:27)

**Comment by marcioadr88** [comment]
I have the same problem on android 7.0 with RN 0.48.4

**Comment by kbarralon** [comment]
I had the same problem. In [this file](https://github.com/crazycodeboy/react-native-splash-screen/blob/master/android/src/main/res/values/refs.xml), there is a reference to a primary_dark color.

So, try to add the primary_color in res/values/colors.xml (create the file if not exists) : 
```
<?xml version="1.0" encoding="utf-8"?>
<resources>
  <color name="primary_dark">#000000</color>
</resources>
```

**Comment by hushicai** [comment]
@kbarralon I solved the problem in the same way.

**Comment by marcioadr88** [comment]
Thaks, @kbarralon. Adding the reference to "primary_dark" solved the problem. I think this should be an optional step.

**Comment by hushicai** [comment]
@marcioadr88 Actually, In most android devices, splash screen works fine.

**Comment by hushicai** [comment]
I had tried it, when downupgrade to 3.0.1, It works fine. but 3.0.6 failed with missing `colors.xml`

**Comment by ismjt** [comment]
@kbarralon Thanks!

**Comment by OrangeFlavoredColdCoffee** [comment]
but  still error  @kbarralon 

Error:Execution failed for task ':react-native-splash-screen:packageReleaseResources'.
> [color/primary_dark] --/node_modules/react-native-splash-screen/android/src/main/res/values/colors.xml	[color/primary_dark] /--/node_modules/react-native-splash-screen/android/src/main/res/values/refs.xml: Error: Duplicate resources

**Comment by christophermark** [comment]
I've put up a PR to fix this (see #125) which will allow custom theming, but doesn't force it. 

It'll fix the BC break introduced in version 3.0.6.
